### ISSUE TITLE
No Connectivity within all local addresses, for whatever reason.. 10.23.91.0/26 = the network and subnet.

### ISSUE THREAD

**Issue by pr3dicament** [Adamm00/IPSet_ASUS#157]
###### Brief Description Of Issue

No connectivity on local addresses.  Entire network becomes locked out.  This happens after the reboot after installing Skynet, usually.  It blocks everything on my own network.

###### Steps To Reproduce Issue

Using a local IP, mine was 10.23.91.30/26 (i do things backwards).. after installing Skynet, I cannot access it at all, at least, once i reboot.  Everything gets blocked, I can't SSH, can't reach it at all, and I end up having to factory reset.  I tried so many things including whitelisting and it would not matter, it would say it's not blocked, can't delete it.. but my IPs were blocked when I looked at the "stats" .. its almost like it blocked basically all IPv4, even local addresses.  Tried adding two networks bridged through ssh... could not get anything to work.  Sometimes wireless connections would work for the LAN, but no internet connectivity.  And it doesn't always work or keep working.   but it automatically banned my wired IPs..  It's not so much reproducing as it is just not being able to use it.  Couldn't whitelist, it kept saying it was not there so that it couldn't delete it, but it was showing it on the stats.  I looked everywhere I could think and it was almost like I couldn't add anything to any whitelist in any way.

TL; DR: I could not whitelist my own local network.  When I did the stats, it showed my WIRED IP's were being blocked.  Made no sense to me.

###### Expected Behaviour

To be able to access the local network, have internet access, and not to have everything blocked.  To be able to use it without issue.  Skynet works great on a GT-AXE11000.  Not on a GT-BE98 Pro.  Is it just me?

###### Output of ( sh /jffs/scripts/firewall debug info )

Router Model; GT-BE98_Pro
Skynet Version; v7.6.3 (27/07/2024) (319a580d2ec003b4e6647fdbf889dc77)
iptables v1.4.15 - (eth0 @ 10.23.91.30)
ipset v7.6, protocol version: 7
IP Address; (x.x.x.x) - (xxxx:xxxx:xxxx:xxxx::/64)
FW Version; 102.2_0 (Nov 3 2024) (4.19.275)
Install Dir; /tmp/mnt/ROUTER-USB3/skynet (96.4G / 112.2G Space Available)
SWAP File; /tmp/mnt/ROUTER-USB3/myswap.swp (10.0G)
Syslog Location; (/jffs/syslog.log) (/jffs/syslog.log-1)
Uptime; 0 days, 3 hours, 15 minutes.
Ram Available; (1151M / 2000M)

---------------                          | ------------              
| Device Name |                          | | Local IP |                            | Status |     
---------------                          | ------------     MAC ADDRESSES REMOVED          

ROUTER-BACKEND                           | 10.23.91.1       | 			    | Inactive
SERVER                                   | 10.23.91.2       |			    | Online
ASUS-DESKTOP                             | 10.23.91.3       | 			    | Online
DVR-Office                               | 10.23.91.4       | 			    | Inactive
DVR-Bedroom                              | 10.23.91.5       |  			    | Inactive
STB-MasterBedroom                        | 10.23.91.6       | 			    | Inactive
Unknown                                  | 10.23.91.7       | Unknown              | Offline
STB-LivingRoom                           | 10.23.91.8       | 			    | DELAY
SPIN3                                    | 10.23.91.9       | 			    | Online
Unknown                                  | 10.23.91.10      | Unknown              | Offline
Unknown                                  | 10.23.91.11      | 		           | Inactive
Unknown                                  | 10.23.91.12      | Unknown              | Offline
Unknown                                  | 10.23.91.13      | Unknown              | Offline
Pixel-8a                                 | 10.23.91.14      | 			   | Inactive
iPkone-Natalie                           | 10.23.91.15      | 			   | Online
iPhone-CD                                | 10.23.91.16      | 			   | Inactive
XBOXONE                                  | 10.23.91.17      | 			   | Online
Google, Inc.                             | 10.23.91.18      | 			   | Online
Pixel-8-Pro                              | 10.23.91.19      | 			   | Inactive
NITRO5                                   | 10.23.91.20      | 			   | Online
Unknown                                  | 10.23.91.21      | Unknown              | Offline
Unknown                                  | 10.23.91.22      | Unknown              | Offline
Unknown                                  | 10.23.91.23      | Unknown              | Offline
Unknown                                  | 10.23.91.24      | Unknown              | Offline
Unknown                                  | 10.23.91.25      | Unknown              | Offline
Google, Inc.                             | 10.23.91.26      | 			   | Inactive
Unknown                                  | 10.23.91.27      | Unknown              | Offline
Unknown                                  | 10.23.91.28      | Unknown              | Offline
Unknown                                  | 10.23.91.29      | Unknown              | Offline
Unknown                                  | 10.23.91.31      | Unknown              | Offline
Unknown                                  | 10.23.91.32      | Unknown              | Offline
Unknown                                  | 10.23.91.33      | Unknown              | Offline
Unknown                                  | 10.23.91.34      | Unknown              | Offline
Chromecast                               | 10.23.91.35      | 			   | Online
Unknown                                  | 10.23.91.36      | Unknown              | Offline
Unknown                                  | 10.23.91.37      | Unknown              | Offline
Unknown                                  | 10.23.91.38      | Unknown              | Offline
Unknown                                  | 10.23.91.39      | Unknown              | Offline
Chromecast-Ultra                         | 10.23.91.40      | 		           | Inactive
Unknown                                  | 10.23.91.41      | Unknown              | Offline
Unknown                                  | 10.23.91.42      | Unknown              | Offline
Unknown                                  | 10.23.91.43      | Unknown              | Offline
Unknown                                  | 10.23.91.44      | Unknown              | Offline
Unknown                                  | 10.23.91.45      | Unknown              | Offline
Unknown                                  | 10.23.91.46      | Unknown              | Offline
Unknown                                  | 10.23.91.47      | Unknown              | Offline
Unknown                                  | 10.23.91.48      | Unknown              | Offline
Unknown                                  | 10.23.91.49      | Unknown              | Offline
Unknown                                  | 10.23.91.50      | Unknown              | Offline
Unknown                                  | 10.23.91.51      | 			   | Inactive
Unknown                                  | 10.23.91.52      | Unknown              | Offline
Unknown                                  | 10.23.91.53      | Unknown              | Offline
Unknown                                  | 10.23.91.54      | Unknown              | Offline
Unknown                                  | 10.23.91.55      | Unknown              | Offline
Unknown                                  | 10.23.91.56      | Unknown              | Offline
Unknown                                  | 10.23.91.57      | Unknown              | Offline
Unknown                                  | 10.23.91.58      | Unknown              | Offline
Unknown                                  | 10.23.91.59      | Unknown              | Offline
Unknown                                  | 10.23.91.60      | Unknown              | Offline
Unknown                                  | 10.23.91.61      | Unknown              | Offline
Unknown                                  | 10.23.91.62      | Unknown              | Offline
Juniper Networks                         | x.x.x.x     | 			   | Online   < --- This is my current IP address, hence the x.x.x.x

--------------------                | ----------
| Test Description |                | | Result |
--------------------                | ----------

Internet-Connectivity               | [Passed]
Public IP Address                   | [Passed]
Write Permission                    | [Passed]
Config File                         | [Passed]
Firewall-Start Entry                | [Passed]
Services-Stop Entry                 | [Passed]
Service-Event Entry                 | [Passed]
Profile.add Entry                   | [Passed]
SWAP File                           | [Passed]
Cron Jobs                           | [Passed]
NTP Sync                            | [Passed]
Log Level 5 Settings                | [Passed]
Duplicate Rules In RAW              | [Passed]
IPSets                              | [Passed]
IPTables Rules                      | [Passed]
Local WebUI Files                   | [Passed]
Mounted WebUI Files                 | [Passed]
MenuTree.js Entry                   | [Passed]

-----------                         | ----------
| Setting |                         | | Status |
----------                          | ----------

Skynet Auto-Updates                 | [Enabled]
Malware List Auto-Updates           | [Enabled]
Logging                             | [Enabled]
Filter Traffic                      | [Enabled]
Unban PrivateIP                     | [Enabled]
Log Invalid Packets                 | [Disabled]
Import AiProtect Data               | [Enabled]
Secure Mode                         | [Enabled]
Fast Switch List                    | [Disabled]
Syslog Location                     | [Default]
IOT Blocking                        | [Disabled]
IOT Logging                         | [Enabled]
Country Lookup For Stats            | [Enabled]
CDN Whitelisting                    | [Enabled]
Display WebUI                       | [Enabled]

18/18 Tests Sucessful              

=============================================================================================================

[#] 35194 IPs (+0) -- 2579 Ranges Banned (+0) || 24 Inbound -- 12 Outbound Connections Blocked! [debug] [3s]

_______________________________________________________________

Obviously I just removed my MAC addresses there.  Otherwise, that's the output of firewall debug

**Comment by Adamm00** [comment]
Skynet logs every packet it blocks, there is never an exception to this rule. Please provide logs of IP's in question being blocked if you believe Skynet is the cause of your issues and we can go from there.

**Comment by pr3dicament** [comment]
Could you please tell me the command to generate a log for you guys?  Also, Skynet works with 192.160.50.0/24 and allows everything local there, but the issue is that I cannot find any logs for 10.23.91.0/26.  I looked everywhere, no dice.. Is there a command I can issue within SSH?  (Yes)  Could you please tell me so I can give you all what you need.  This is my fourth time re-writing this comment since i broke my wrist, so apologies for any typos.

You could issue a command that pulls this information.  We both know that.  I have not been lazy with this, I just simply removed SkyNet and things started workin.  It blocked my entire local network and from what i can tell, didnt generate any logs,  I couldn't do ANYTHING with its WebUI too, even SSH was limited.  local basically was crippled so i had to factory reset the router.  I'm trying all of this  stuff, left and right, i'm literally having comerse wouldn't work either.  Both v4 and v6.

Sorry.  too long of a response.  tl;dr There aren't any logs that i can find.
LAN IP: 10.23.91.62
Subnet Mask: 255.255.255.192

**Comment by Adamm00** [comment]
> Sorry. too long of a response. tl;dr There aren't any logs that i can find.

Skynet logs everything to the syslog in real time, then cleans up the syslog to its permanent location in “/tmp/mnt/ROUTER-USB3/skynet” every hour. You can also use the various stat commands in the readme to identify any blocks you believe to be incorrect.
